src/main/java/uk/gov/companieshouse/api/accounts/validation/CurrentPeriodValidator.java

```diff
@@ -0,0 +1,45 @@
+package uk.gov.companieshouse.api.accounts.validation;
```

Comment:
Should probably have an interface for this class. Possibly generic so `public interface Validator<C extends RestObject>` with a single method called `Errors validate(@Valid C rest, Errors errors);` not 100% sure though but certainly warrants a discussion.

Reply:
It has been suggested we leave it as it is for now. We can always implement an interface at a later date if it is needed.